socketio/asyncio_aiopika_manager.py

```diff
@@ -80,16 +82,7 @@ async def _listen(self):
         while True:
             try:
                 if self.listener_connection is None:
-                    self.listener_connection = await self._connection()
-                    self.listener_channel = await self._channel(
-                        self.listener_connection
-                    )
-                    await self.listener_channel.set_qos(prefetch_count=1)
-                    exchange = await self._exchange(self.listener_channel)
-                    self.listener_queue = await self._queue(
-                        self.listener_channel, exchange
-                    )
-                    retry_sleep = 1
+                    await self._initialize()
```

Comment:
This is problematic, because you have one more _initialize() call above. Both will set the same variables.